conda/recipes/libcuspatial/meta.yaml

```diff
@@ -34,18 +34,35 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
     - {{ compiler('cxx') }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} {{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }} {{ cuda_version }}
+    - cuda-cudart ={{ cuda_version }}
+    - cuda-cudart-dev ={{ cuda_version }}
```

Comment:
Delete all the cuda-cudart* packages from build and add **only** cuda-cudart-dev to host.